src/www/ui_components/app-root.js

```diff
@@ -524,6 +526,11 @@ export class AppRoot extends mixinBehaviors([AppLocalizeBehavior], PolymerElemen
         type: String,
         readonly: true,
       },
+      isElectron: {
```

Comment:
*minor* Is the name `isElectron` too specific about the implementation details? Maybe call it `hideFeedback`? What do you think?